### PR TITLE
refactoring installation. hook is now just a link into commitable code

### PR DESCRIPTION
**Pull request by pschrammel**
Hi,

I changed the way the hook in .git/hooks is created. now it's just a link to .codeqa/git_hook.rb . The script is then executing everything in .codeqa/hooks . This way hooks in any language can be written and hooks can be commited with the project. Please release if you accept my changes. Thx.

### CODE REVIEW

**Comment by michaelsauter** [comment]
Could you look into the test failures as well? Would be great to get those green.

File: .rubocop.yml [hunk 1]
```diff
@@ -3,6 +3,23 @@ AllCops:
...
+  Enabled: false
+
+
+Metrics/MethodLength:
+  Enabled: false
+  CountComments: false # count full line comments?
+  Max: 10
+  Severity: refactor
+
+
```
Comment (michaelsauter) [inline .rubocop.yml]: Why were these moved from the disabled section below?

**Comment by michaelsauter** [comment]
Overall, I'd like to merge this ASAP. Thanks for the PR!

**Comment by pschrammel** [comment]
Okay, things should work now. Rewerted the changes to .rubocop.yml but some files don't comply to it. I'd cleanup the code style issues in a separate pull request and let travis check for compliance.

**Comment by michaelsauter** [comment]
Thanks! :+1:
